.gitignore

```diff
@@ -1 +1,19 @@
+.gradle
```

Comment:
I would recommend also providing:
* editorconfig file
* Jenkinsfile
* travis ci file
* codecoverage example

Reply:
All of these are going to be added as a part of the story, but as I mentioned in the description, this is the very basic one. I don't want to add everything in one pull request.